### PR TITLE
Festival overview & Festival detail page

### PR DESCRIPTION
**Pull request by davidwobido**
## ✅  Done
- [x] All information comes from API
- [x] Name of clicked festival on SmallFestivalCard-Component is passed to AllFestivals-page …
- [x] … and passed to FestivalCardLarge so the right festival is displayed
- [x] Add close-function close-button to FestivalCardLarge
- [x] Page should not rerender when close-button is clicked

## 🚧  TBD
CSS: Class _Container_ will be deleted when page is implemented 

## 🎉 Nice to have if there is some time left
• left column bold or medium
• back to top button
• Search bar should always have same size as the width of the FestivalCardSmall rows

https://user-images.githubusercontent.com/91462450/145705032-b528af3a-0d71-40ef-bd87-492ee1272c64.mov

### CODE REVIEW

File: src/components/FestivalCardLarge/FestivalCardLarge.tsx [hunk 1]
```diff
@@ -1,38 +1,67 @@
...
+export type FestivalCardLargeProps = {
+  name: string;
+  location: string;
+  begin: string;
+  end: string;
+  visitors: number;
+  acts: number;
+  price: number;
+  allacts: string;
+  website: string;
```
Comment (F-Kirchhoff) [inline src/components/FestivalCardLarge/FestivalCardLarge.tsx:12]: you could wrap these props into a content object, then you can simply hand over the Festival object in a single prop

File: src/pages/AllFestivals/AllFestivals.tsx [hunk 2]
```diff
@@ -22,28 +27,71 @@ function AllFestivals() {
...
+        setFestival(body);
+      }
+    }
+    clickedFestival();
+  }, [query]);
+
+  function close() {
+    setFestival("");
+  }
+
```
Comment (F-Kirchhoff) [inline src/pages/AllFestivals/AllFestivals.tsx:44]: Isn't the data for the festival also stored in the festivals array?

**Review by F-Kirchhoff — APPROVED** [review]
very nice! you could wrap the props of your cards into a content object for better readability and maybe avoid the second fetch if an festival is displayed in the large card but aside that it's top.

File: src/pages/AllFestivals/AllFestivals.tsx (hunk 2 above)
Comment (davidwobido) [reply src/pages/AllFestivals/AllFestivals.tsx:44]: Oh yes, you’re absolutely right! I’ll check this out, thanks!

**Pull request merged**